src/OSPSuite.UI/Views/Importer/ColumnMappingControl.cs

```diff
@@ -115,31 +118,32 @@ public override void InitializeBinding()
 
          _gridViewBinder.AutoBind(x => x.Description)
             .WithCaption(Captions.Description)
-            .WithRepository(valueRepository)
-            .WithOnValueUpdating(onValueChanged)
+            .WithRepository(descriptionRepository)
+            .WithOnValueUpdated((o, e) => onValueChanged(o))
             .WithShowButton(ShowButtonModeEnum.ShowAlways);
 
          _gridViewBinder.AddUnboundColumn()
             .WithCaption(UIConstants.EMPTY_COLUMN)
-            .WithShowButton(ShowButtonModeEnum.ShowOnlyInEditor)
+            .WithShowButton(ShowButtonModeEnum.ShowAlways)
             .WithRepository(removeRepository)
             .WithFixedWidth(UIConstants.Size.BUTTON_WIDTH);
 
          _gridViewBinder.AddUnboundColumn()
             .WithCaption(UIConstants.EMPTY_COLUMN)
-            .WithShowButton(ShowButtonModeEnum.ShowOnlyInEditor)
+            .WithShowButton(ShowButtonModeEnum.ShowAlways)
             .WithRepository(unitRepository)
             .WithFixedWidth(UIConstants.Size.BUTTON_WIDTH);
 
          _gridViewBinder.AddUnboundColumn()
             .WithCaption(UIConstants.EMPTY_COLUMN)
-            .WithShowButton(ShowButtonModeEnum.ShowOnlyInEditor)
+            .WithShowButton(ShowButtonModeEnum.ShowAlways)
             .WithRepository(lloqRepository)
             .WithFixedWidth(UIConstants.Size.BUTTON_WIDTH);
 
          _removeButtonRepository.ButtonClick += (o, e) =>
          {
             columnMappingGridView.ActiveEditor.EditValue = ColumnMappingFormatter.Ignored();
+            columnMappingGridView.CloseEditor();
```

Comment:
This ensures that the editor is released. I am not sure why this is required in the first place, probably because of how the objects are updated?  not sure